backend/clap/src/main/java/zpi/ppea/clap/mails/EmailServiceImpl.java

```diff
@@ -0,0 +1,95 @@
+package zpi.ppea.clap.mails;
+
+import data_store.EmailDetails;
+import jakarta.mail.MessagingException;
+import jakarta.mail.internet.MimeMessage;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Service;
+import org.springframework.util.FileCopyUtils;
+import zpi.ppea.clap.config.ValueConfig;
+import zpi.ppea.clap.repository.EmailRepository;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+@Slf4j
+@Service
+@AllArgsConstructor
+public class EmailServiceImpl {
+
+    private final JavaMailSender emailSender;
+    private final EmailRepository emailRepository;
+    private final ValueConfig valueConfig;
+
+    public void sendHtmlMessageFromFile() {
+        var emailResponse = emailRepository.getEmailsToSent();
+        if (emailResponse == null)
+            return;
+
+        var emailsList = emailResponse.getEmailsList();
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ssX");
+
+        for (var email : emailsList) {
+            LocalDateTime ratingDateTime = LocalDateTime.parse(email.getRatingSubmitDate(), formatter);
```

Comment:
I would move this logic to datastore. DataStore already does this so including it as a field in `proto` would be easy.
This way Clap doesn't need to parse the time and just treat it as a string - we can avoid worrying about time zones, time formats and other things when it comes to that.